docs/apps/elmer.md

```diff
@@ -97,4 +99,4 @@ It can be advantageous to utilize fewer cores per node as available, which can i
 
```

Comment:
### Example parallel batch script for LUMI

```bash
#!/bin/bash 
#SBATCH --time=06:00:00
#SBATCH --job-name=jobname
#SBATCH --output=%x.%j.out
#SBATCH --error=%x.%j.err
#SBATCH --partition=small
#SBATCH --account=<project>
#SBATCH --nodes=4
#SBATCH --ntasks=128

module load elmer/latest
# make sure the SIF is in the start-info
echo myrun.sif > ELMERSOLVER_STARTINFO
echo "starting Elmer simulation with SIF file"
cat ELMERSOLVER_STARTINFO
srun ElmerSolver
echo "done"
```

Reply:
This is a good example of a run where we undersubscribe a node, similarly to the example above with Mahti. If we don't want to explicitly undersubscribe the nodes in this example, then we can switch `--ntasks=128` to `--ntasks-per-node=128` here.